src/main/java/com/vestmark/bitbucket/plugin/VersionComparator.java

```diff
@@ -41,8 +41,8 @@ public VersionComparator(Function<T, String> valueMapper, String splitPattern)
 
   public int compare(T o1, T o2)
   {
-    String[] l = valueMapper.apply(o1).split(splitPattern);
-    String[] r = valueMapper.apply(o2).split(splitPattern);
+    String[] l = valueMapper.apply(o1).replaceAll("release\\/","").split(splitPattern);
```

Comment:
This logic was lifted from another, internal library used primarily by the auditing tools.
That library needs the same treatment.